webapp/scripts/functions.js

```diff
@@ -109,7 +120,7 @@ module.exports = {
    */
   getUnivarBounds(mapset, map, column) {
     const stats = module.exports.getUnivar(mapset, map, column)
-    return [round(stats.min, 2), round(stats.max, 2)]
+    return stats != undefined ? [round(stats.min, 2), round(stats.max, 2)] : []
```

Comment:
This is equivalent to `stats ? <expression> : <expression>`. If you want to test for `undefined` instead of falsy, use `!==`